lib/t_web/views/view_helpers.ex

```diff
@@ -69,78 +51,70 @@ defmodule TWeb.ViewHelpers do
         if Map.has_key?(label, "text-contact") do
           acc
         else
-          [process_label(label) | acc]
+          case process_label(label, version) do
+            nil -> acc
+            label -> [label | acc]
+          end
         end
       end)
       |> :lists.reverse()
 
     %{page | "labels" => labels}
   end
 
-  defp process_labels_v6(page), do: page
+  defp process_labels(page, _version), do: page
 
-  defp process_labels_pre_v6(%{"labels" => labels} = page) do
-    labels =
-      labels
-      |> Enum.reduce([], fn label, acc ->
-        # users on version < 6.0.0 don't support contact stickers, so they are removed
-        if Map.get(label, "question") in Profile.contacts() or Map.get(label, "text-change") do
-          acc
-        else
-          label = label |> process_label() |> Map.delete("text-contact")
-          [label | acc]
-        end
-      end)
-      |> :lists.reverse()
-
-    %{page | "labels" => labels}
-  end
-
-  defp process_labels_pre_v6(page), do: page
-
-  defp process_label(%{"question" => "telegram", "answer" => handle} = label) do
+  defp process_label(%{"question" => "telegram", "answer" => handle} = label, _version) do
```

Comment:
process_label shouldn't have `_version` if it's being ignored in every but one function clause